File: python-client-sdk/src/sdk/models/shared/projectrunresponsepayload.py
from __future__ import annotations
import dataclasses
from dataclasses_json import Undefined, dataclass_json
from sdk import utils


@dataclass_json(undefined=Undefined.EXCLUDE)
@dataclasses.dataclass
class ProjectRunResponsePayload:
    r"""ProjectRunResponsePayload
    Response format returned by the runProject endpoint
    """
    
    project_id: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('projectId') }})
    project_version: int = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('projectVersion') }})
    run_id: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('runId') }})
    run_status_url: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('runStatusUrl') }})
    run_url: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('runUrl') }})
    trace_id: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('traceId') }})
    

File: java-client-sdk/lib/src/main/java/org/openapis/openapi/models/shared/ProjectRunResponsePayload.java
package org.openapis.openapi.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * ProjectRunResponsePayload
 * Response format returned by the runProject endpoint
**/
public class ProjectRunResponsePayload {
    @JsonProperty("projectId")
    public String projectId;
    public ProjectRunResponsePayload withProjectId(String projectId) {
        this.projectId = projectId;
        return this;
    }
    
    @JsonProperty("projectVersion")
    public Integer projectVersion;
    public ProjectRunResponsePayload withProjectVersion(Integer projectVersion) {
        this.projectVersion = projectVersion;
        return this;
    }
    
    @JsonProperty("runId")
    public String runId;
    public ProjectRunResponsePayload withRunId(String runId) {
        this.runId = runId;
        return this;
    }
    
    @JsonProperty("runStatusUrl")
    public String runStatusUrl;
    public ProjectRunResponsePayload withRunStatusUrl(String runStatusUrl) {
        this.runStatusUrl = runStatusUrl;
        return this;
    }
    
    @JsonProperty("runUrl")
    public String runUrl;
    public ProjectRunResponsePayload withRunUrl(String runUrl) {
        this.runUrl = runUrl;
        return this;
    }
    
    @JsonProperty("traceId")
    public String traceId;
    public ProjectRunResponsePayload withTraceId(String traceId) {
        this.traceId = traceId;
        return this;
    }
    
}
